src/style.scss

```diff
@@ -82,6 +83,7 @@
 
     &--ondark {
       &:hover {
+        // This somehow doesn't refer to anywhere
         color: var(--auro-hyperlink-ondark-hover-color);
```

Comment:
This new `--ds-` token doesn't exist.

We should figure out what the old token resolved to for the value and then find a token in the new DS tokens with the same value.

Reply:
Agree, and I'd love to do that. Problem is I couldn't find the old reference for `--auro-hyperlink-ondark-hover-color`.
Not sure if this token name refers to this [figma](https://www.figma.com/file/TSFOFDwWQxmpPpYQ2F06lH/Hyperlink?type=design&node-id=322-813&mode=design&t=IXUlR4ZVKsSZPSZS-0), or there is custom token that specifically created for hyperlink only.

<img width="512" alt="image" src="https://github.com/AlaskaAirlines/auro-hyperlink/assets/124668226/9024dad2-0902-4ad7-b8ea-ccfa2361a640">

If it is, then maybe the real token name is `--auro-color-text-primary-ondark` or `--ds-color-text-primary-default` based on the blueprint.

Please advise, thank you.

Reply:
@leeejune can you help us figure out what token should be used here?

Reply:
The real token name for `--auro-color-text-primary-on-light` is `--ds-color-text-primary-default`. The `-on-light` has been converted to `-default` and `-on-dark` is now `-inverse`

Reply:
@fajar-apri-alaska talked with @leeejune directly and it seems we should go ahead and use `--auro-color-ui-hover-on-dark` here.